frontend-react/e2e/spec/all/support-page.spec.ts

```diff
@@ -1,39 +1,87 @@
-import { expect, test } from "@playwright/test";
-import * as support from "../../pages/support";
-// eslint-disable-next-line playwright/no-skipped-test
-test.describe.skip("Support page", () => {
-    test.beforeEach(async ({ page }) => {
-        await support.goto(page);
-    });
+import fs from "fs";
+import path from "path";
+import { SupportPage } from "../../pages/support.js";
+import { test as baseTest, expect } from "../../test";
+
+const cards = [
+    {
+        name: "Welcome to ReportStream",
+        anchorID: "welcome-to-reportstream-1",
+    },
+    {
+        name: "Getting started",
+        anchorID: "getting-started-1",
+    },
+    {
+        name: "Using ReportStream",
+        anchorID: "using-reportstream-1",
+    },
+];
+
+export interface SupportPageFixtures {
+    supportPage: SupportPage;
+}
+
+const test = baseTest.extend<SupportPageFixtures>({
+    supportPage: async (
+        {
+            page: _page,
+            isMockDisabled,
+            adminLogin,
+            senderLogin,
+            receiverLogin,
+            storageState,
+            isFrontendWarningsLog,
+            frontendWarningsLogPath,
+        },
+        use,
+    ) => {
+        const page = new SupportPage({
+            page: _page,
+            isMockDisabled,
+            adminLogin,
+            senderLogin,
+            receiverLogin,
+            storageState,
+            isFrontendWarningsLog,
+            frontendWarningsLogPath,
+        });
+        await page.goto();
+        await use(page);
+    },
+});
 
-    test("should have correct title", async ({ page }) => {
-        await expect(page).toHaveURL(/support/);
-        await expect(page).toHaveTitle(/ReportStream support/);
+test.describe("Support page", () => {
+    test("Should have a way of contacting support", async ({ supportPage }) => {
+        // Importing the actual URL of our contact form to match
+        // what we use in production and only need to make updates
+        // in a single place
+        const jsonPath = path.resolve("./src/content/site.json");
```

Comment:
You can use the modern way of importing a json file in node: `import site from "../../../src/content/site.json" assert { type: "json" };`

Reply:
didn't seem to work on the first go around for some reason....